blocks/DropArea/DropArea.js

```diff
@@ -17,13 +18,28 @@ export class DropArea extends UploaderBlock {
       },
       onFiles: (files) => {
         files.forEach((/** @type {File} */ file) => {
-          this.uploadCollection.add({
-            file,
-            isImage: file.type.includes('image'),
-            mimeType: file.type,
-            fileName: file.name,
-            fileSize: file.size,
-          });
+          let hasValidationErrors = false;
+          let isImage = file.type.includes('image');
```

Comment:
minor
let isImage => const isImage

Reply:
We don't use "prefer const" convention for some reason.